app/src/main/res/drawable/ic_settings.xml

```diff
@@ -8,6 +8,9 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:pathData="M20.8 11h-2.9a6 6 0 0 0-1-2.5l2-2a1 1 0 0 0-1.4-1.4l-2 2a6 6 0 0 0-2.5-1V3.3a1 1 0 0 0-2 0V6a6 6 0 0 0-2.5 1l-2-2a1 1 0 0 0-1.4 1.4l2 2a6 6 0 0 0-1 2.5H3.3a1 1 0 0 0 0 2H6a6 6 0 0 0 1 2.5l-0.2 0.2-1.8 1.8a1 1 0 1 0 1.4 1.4L8.3 17l0.2-0.2a6 6 0 0 0 2.5 1v2.9a1 1 0 0 0 2 0v-2.9a6 6 0 0 0 2.5-1l2 2a1 1 0 0 0 1.4-1.4l-2-2a6 6 0 0 0 1-2.5h2.9a1 1 0 0 0 0-2zM8 12a4 4 0 1 1 8 0 4 4 0 0 1-8 0z"
-      android:fillColor="?primaryText" />
+      android:pathData="M12.8,22h-1.6a2,2 0,0 1,-1.9 -1.4l-0.5,-1.8 -1,-0.6 -1.9,0.4a2,2 0,0 1,-2.2 -1L3,16.4a2,2 0,0 1,0.3 -2.4l1.3,-1.3a6.7,6.7 0,0 1,0 -1.2l-1.3,-1.3A2,2 0,0 1,3 7.7l0.8,-1.4a2,2 0,0 1,2.2 -1l1.9,0.5 1,-0.6 0.5,-1.8a2,2 0,0 1,2 -1.4h1.5a2,2 0,0 1,2 1.4l0.4,1.8 1,0.6 1.9,-0.4a2,2 0,0 1,2.2 1l0.8,1.3a2,2 0,0 1,-0.3 2.4l-1.3,1.3a6.6,6.6 0,0 1,0 1.2l1.3,1.3c0.6,0.7 0.7,1.7 0.3,2.4l-0.8,1.4a2,2 0,0 1,-2.2 1l-1.8,-0.5 -1,0.6 -0.6,1.8a2,2 0,0 1,-2 1.4zM7.8,16.6l0.6,0.2 1.3,0.7 0.4,0.5 0.6,2.1c0.1,0.3 0.3,0.4 0.5,0.4h1.6c0.2,0 0.4,-0.1 0.5,-0.4l0.6,-2 0.4,-0.6a6,6 0,0 0,1.3 -0.7l0.7,-0.2 2.1,0.6c0.2,0 0.5,0 0.6,-0.3l0.8,-1.3 -0.1,-0.6 -1.6,-1.7 -0.2,-0.6a5.8,5.8 0,0 0,0 -1.5l0.2,-0.6L19.7,9c0.1,-0.1 0.2,-0.4 0,-0.6L19,7.1a0.5,0.5 0,0 0,-0.6 -0.3l-2.2,0.6 -0.6,-0.2a6,6 0,0 0,-1.3 -0.7L14,6l-0.6,-2.1c0,-0.3 -0.3,-0.4 -0.5,-0.4h-1.6c-0.2,0 -0.4,0.1 -0.5,0.4l-0.6,2 -0.4,0.6a6,6 0,0 0,-1.3 0.7l-0.6,0.2 -2.2,-0.6c-0.2,0 -0.5,0 -0.6,0.3l-0.8,1.3 0.1,0.6L6,10.6l0.2,0.6a6.1,6.1 0,0 0,0 1.5l-0.2,0.7L4.3,15c-0.1,0.1 -0.2,0.4 0,0.6l0.7,1.3c0.1,0.2 0.4,0.3 0.6,0.3l2.1,-0.6z"
+      android:fillColor="?primaryText"/>
```

Comment:
I also wonder if primaryText is equivalent to mozac_ui_icons_fill. Maybe note this as a potential investigation in the above issue.e

Reply:
No it's not equivalent, [primaryText](
https://github.com/mozilla-mobile/fenix/blob/8c0f86bd96f7eae4d5cfb7af09eaa770aff06837/app/src/main/res/values/attrs.xml#L7) is an attribute (variable) that we use for changing colors for theming ([dark/light](https://github.com/mozilla-mobile/fenix/blob/8c0f86bd96f7eae4d5cfb7af09eaa770aff06837/app/src/main/res/values/styles.xml#L42)) purpose, you can find more info [here](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme).